### DISCUSSION TITLE
Engine speed fluctuations

### DISCUSSION THREAD

**Discussion by vaclavikzahradka** [Mogli12/KeyboardSteer#810] [Ideas]
Hello. On tractors with a manual transmission, the engine speed and engine noise fluctuate at very low speeds. It does so on a small scale, but it's annoying. It's as if the tractor has a malfunctioning regulator in the injection pump. The latest version where it was fine is 9.0.4.3. Everyone else already has this problem. At higher tractor speeds, the engine speed is ok. 

Hallo. Bei Traktoren mit Schaltgetriebe schwanken Motordrehzahl und Motorgeräusch bei sehr niedrigen Drehzahlen. Es tut dies in einem kleinen Maßstab, aber es ist ärgerlich. Es ist, als hätte der Traktor einen defekten Regler in der Einspritzpumpe. Die neueste Version, in der es in Ordnung war, ist 9.0.4.3. Alle anderen haben dieses Problem schon. Bei höheren Traktorgeschwindigkeiten ist die Motordrehzahl in Ordnung.

**Comment by Mogli12** [comment]
Das ist die Leerlaufanreicherung, die ist nur aktiv wenn die Drehzahl eigentlich zu niedrig ist. Du kannst die Motormodifikationen ausschalten.

**Comment by vaclavikzahradka** [comment]
I know what you're talking about, it's okay, but that's not what I'm writing about. This problem occurs at any engine speed up to a speed of approx. 8 km / h, when a lower gear is engaged. If, for example, you have a 2 LL gear on the Valtra (or any tractor with a manual transmission), the speed fluctuates while maintaining speed (1000 to 1900 RPM). Continuous speed is only when the maximum engine speed is reached.
When shifting to a higher gear (1H), it does not, the speed is smooth throughout. 

Ich weiß, wovon du sprichst, es ist okay, aber das ist nicht das, worüber ich schreibe. Dieses Problem tritt bei jeder Motordrehzahl bis zu einer Geschwindigkeit von ca. 8 km/h auf, wenn ein niedrigerer Gang eingelegt wird. Wenn Sie beispielsweise einen 2 LL-Gang auf dem Valtra (oder jedem Traktor mit Schaltgetriebe) haben, schwankt die Geschwindigkeit, während Sie die Geschwindigkeit beibehalten (1000 bis 1900 U / min). Kontinuierliche Geschwindigkeit ist nur, wenn die maximale Motordrehzahl erreicht ist.
Beim Schalten in einen höheren Gang (1H) nicht, die Geschwindigkeit ist durchgehend gleichmäßig.
